charts/velero/README.md

```diff
@@ -28,169 +32,40 @@ A Helm chart for Kubernetes
-| velero.readinessProbe.timeoutSeconds | int | `5` |  |
-| velero.resources | object | `{}` |  |
-| velero.schedules | object | `{}` |  |
-| velero.secretAnnotations | object | `{}` |  |
-| velero.serviceAccount.server.annotations | string | `nil` |  |
-| velero.serviceAccount.server.create | bool | `true` |  |
-| velero.serviceAccount.server.imagePullSecrets | list | `[]` |  |
-| velero.serviceAccount.server.labels | string | `nil` |  |
-| velero.serviceAccount.server.name | string | `nil` |  |
-| velero.snapshotsEnabled | bool | `true` |  |
-| velero.terminationGracePeriodSeconds | int | `3600` |  |
-| velero.tolerations | list | `[]` |  |
-| velero.upgradeCRDs | bool | `true` |  |
-| velero.upgradeCRDsJob.extraEnvVars | object | `{}` |  |
-| velero.upgradeCRDsJob.extraVolumeMounts | list | `[]` |  |
-| velero.upgradeCRDsJob.extraVolumes | list | `[]` |  |
-| velero.upgradeJobResources | object | `{}` |  |
+| backups.enabled | bool | `false` | Enables creation of a velero schedule |
+| backups.namespaces | list | `["default"]` | List of namespace backuped by velero |
+| backups.schedule | string | `"@every 24h"` | Period of backup |
+| backups.ttl | string | `"96h0m0s"` | ttl of the backup |
+| gcpbucket | object | `{"accessControl":{"createCloudIamPolicy":true,"iamPolicy":[{"member":"serviceAccount:sa-name@${gcpProjectId?}.iam.gserviceaccount.com","role":"roles/storage.admin"}],"publicAccessPrevention":"inherited","uniformBucketLevelAccess":false},"bucketName":"","global":{"abandon":false,"cnrmNamespace":"","gcpProjectId":"myprojectid","location":"EUROPE-WEST1","skipUnspecifiedFields":false}}` | `tags.configConnector` must be set to `true` |
+| iamCustomRole | object | `{"customRoleName":"velero","description":"The description of the custom role resource","global":{"cnrmNamespace":"","gcpOrganisationId":"","gcpProjectId":"myprojectid","skipUnspecifiedFields":false},"permissions":["compute.disks.create","compute.disks.createSnapshot","compute.disks.get","compute.snapshots.create","compute.snapshots.delete","compute.snapshots.get","compute.snapshots.useReadOnly","compute.zones.get"],"title":"velero"}` | `tags.configConnector` must be set to `true` |
+| iamCustomRole.customRoleName | string | `"velero"` | The name of the IAM Custom Role |
+| iamCustomRole.description | string | `"The description of the custom role resource"` | A human-readable description for the role |
+| iamCustomRole.permissions | list | `["compute.disks.create","compute.disks.createSnapshot","compute.disks.get","compute.snapshots.create","compute.snapshots.delete","compute.snapshots.get","compute.snapshots.useReadOnly","compute.zones.get"]` | custom role permissions |
+| iamCustomRole.title | string | `"velero"` | A human-readable title for the role |
+| iamPolicyMembers | object | `{"global":{"abandon":false,"cnrmNamespace":"","gcpProjectId":"myprojectid"},"iamPolicyMembers":[]}` | `tags.configConnector` must be set to `true` |
```

Comment:
`iamPolicyMembers` should respect the schema from the `gcp-iam-policymembers` chart.

Now the values are like this:

```yaml
iamPolicyMembers:
  ...
  members: []
```

take a look at the `gcp-iam-policymembers` please.